sway-lsp/src/core/session.rs

```diff
@@ -71,6 +74,8 @@ pub struct Session {
     parse_permits: Arc<Semaphore>,
     // Cached diagnostic results that require a lock to access. Readers will wait for writers to complete.
     diagnostics: Arc<RwLock<Diagnostics>>,
+    // Enables the experimental GC pass, disabled for now.
+    enable_gc: bool,
```

Comment:
Since this has no guarantee to work at this point I think we should add "experimental" to the name of the config field so it's clear in the call as well.